runtime/polkadot/src/lib.rs

```diff
@@ -1247,46 +1172,55 @@ sp_api::impl_runtime_apis! {
 			Executive::offchain_worker(header)
 		}
 	}
-
-	impl parachain::ParachainHost<Block> for Runtime {
-		fn validators() -> Vec<parachain::ValidatorId> {
-			Parachains::authorities()
+	// Dummy implementation to continue supporting old parachains runtime temporarily.
+	impl p_v0::ParachainHost<Block> for Runtime {
+		fn validators() -> Vec<p_v0::ValidatorId> {
+			// this is a compile-time check of size equality. note that we don't invoke
+			// the function and nothing here is unsafe.
+			let _ = core::mem::transmute::<p_v0::ValidatorId, AccountId>;
```

Comment:
can we use https://docs.rs/static_assertions/1.1.0/static_assertions/macro.assert_eq_size.html instead?

Reply:
Why add a crate when `core` can do it?

Reply:
This is a stop-gap measure anyway, only needed for one runtime upgrade. I worry that we would forget to remove the crate later on

Reply:
the intent is more clear with `static_assertions` and it doesn't require an additional comment to explain it, but yeah, ignore my grumble as its a stop-gap :)